### PR TITLE
Use latest version of plugin lib functionality

### PR DESCRIPTION
**Pull request by tboerger**
This boilr template is also used for https://github.com/drone-plugins/drone-curseforge now.

### CODE REVIEW

File: template/.drone.star
Comment (donny-dont) [inline template/.drone.star:58]: I still feel like if we have a major minor golang version that printing out the actual version is potentially useful information.

Honestly maybe even a `go env` might be a good thing to have when building.

**Review by donny-dont — APPROVED** [review]

File: template/.drone.star
Comment (tboerger) [reply template/.drone.star:58]: Still not entirely sure about the factor to be really useful. For the windows builds it would make sense because we can not even see which version gets used as it's not managed by docker images.

File: template/.gitignore
Comment (donny-dont) [inline template/.gitignore:4]: I feel like the `.drone.yml` should also be added here in case someone is running it through `drone exec` to see what things look like locally

File: template/.drone.star
Comment (donny-dont) [reply template/.drone.star:58]: If the ssh runners have it that's fine by me for a compromise.

Just a heads up I don't know that @bradrydzewski updated the SSH runners for windows to be Go 1.13

Comment (tboerger) [reply template/.drone.star:58]: I have added `go version` and `go env` calls to the linux and windows pipelines, there you go :)

File: template/.gitignore
Comment (tboerger) [reply template/.gitignore:4]: Fixed via https://github.com/drone-plugins/boilr-plugin/pull/6/commits/b7fe98ac474559ceaccb09dfb732e950f6d32649

**Comment by tboerger** [comment]
@donny-dont maybe you want to do another review? After merging that I will apply the changes to drone-curseforge.

**Pull request merged**
